### ISSUE TITLE
lifelines.datasets not importing properly?

### ISSUE THREAD

**Issue by ApproximateIdentity** [CamDavidsonPilon/lifelines#140]
I'm going through the docs on this page http://lifelines.readthedocs.org/en/latest/Intro%20to%20lifelines.html

Running the following code on the master branch fails for me:

```python
import pandas as pd
import lifelines

data = lifelines.datasets.load_dd()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-0ed721d6a308> in <module>()
----> 1 data = lifelines.datasets.load_dd()

AttributeError: 'module' object has no attribute 'datasets'
```

I made made a oneline change to fix this which you can see here: https://github.com/ApproximateIdentity/lifelines/commit/6ea572198138fd49bf9782cba51401381331330f

This fixes the problem for me. I could do a pull request, but maybe you don't want it to load in this fashion?

**Comment by CamDavidsonPilon** [comment]
Hmm, I don't know what the right way to correct this is... @spacecowboy any ideas?

**Comment by spacecowboy** [comment]
Depends on if you want to support the API that @ApproximateIdentity is referring to. If you do, his fix is correct.

As the package is currently structured, correct usage at this moment would be one of:

    from lifelines import datasets
    
    data = datasets.load_dd()

more direct:

    from lifelines.datasets import load_dd
    
    data = load_dd()

**Comment by CamDavidsonPilon** [comment]
Add in https://github.com/CamDavidsonPilon/lifelines/commit/cf8b180a2e9a0dce844e8412f23f6c99e8fa3ece

**Comment by juancq** [comment]
This is still an issue in version 0.27.0, python 3.10.6. 
```python
>>> import lifelines
>>> lifelines.datasets
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'lifelines' has no attribute 'datasets'
>>> lifelines.__version__
'0.27.0'
```

**Comment by CamDavidsonPilon** [comment]
This should be fixed in 0.27.7 just released
